subprojects/internal-integ-testing/internal-integ-testing.gradle.kts

```diff
@@ -40,7 +40,10 @@ dependencies {
     compile(library("jansi"))
     compile(library("commons_collections"))
     compile("org.apache.mina:mina-core")
-    compile(testLibrary("sampleCheck"))
+    compile(testLibrary("sampleCheck")) {
+        // asciidoctorj depends on a lot of stuff, which causes `Can't create process, argument list too long` on Windows
+        exclude(group = "org.asciidoctor", module = "asciidoctorj")
```

Comment:
Since we use that logic now in two places, can we use a `ComponentMetadataRule` instead? So everyone depending on `sampleCheck` in the Gradle build would automatically get the right dependencies?